src/AspNet.Security.OAuth.JumpCloud/JumpCloudAuthenticationOptions.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.JumpCloud;
+
+/// <summary>
+/// Defines a set of options used by <see cref="JumpCloudAuthenticationHandler"/>.
+/// </summary>
+public class JumpCloudAuthenticationOptions : OAuthOptions
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="JumpCloudAuthenticationOptions"/> class.
+    /// </summary>
+    public JumpCloudAuthenticationOptions()
+    {
+        ClaimsIssuer = JumpCloudAuthenticationDefaults.Issuer;
+        CallbackPath = JumpCloudAuthenticationDefaults.CallbackPath;
+
+        Scope.Add("openid");
+        Scope.Add("profile");
+        Scope.Add("email");
+
+        ClaimActions.MapJsonKey(ClaimTypes.Email, "email");
+        ClaimActions.MapJsonKey(ClaimTypes.GivenName, "given_name");
+        ClaimActions.MapJsonKey(ClaimTypes.Name, "name");
+        ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "sub");
+        ClaimActions.MapJsonKey(ClaimTypes.Surname, "family_name");
+    }
+
+    /// <summary>
+    /// Gets or sets the JumpCloud domain (Org URL) to use for authentication.
+    /// </summary>
+    public string? Domain { get; set; }
```

Comment:
Does JumpCloud support configuring your own domain?
  - If not, we shouldn't make the domain configurable and remove this property.
  - If so, it would likely make sense to use `https://oauth.id.jumpcloud.com/` as the default domain when the user doesn't specify an explicit value.